package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
     "test": "jest",
-    "lint": "tslint -t stylish 'src/**/*.ts'"
+    "lint": "tslint -t stylish 'src/**/*.ts'",
+    "build:watch": "tsc --watch",
+    "serve:watch": "nodemon --ext js --delay 250ms build/server.js",
+    "lint:watch": "chokidar \"**/*.ts\" -c \"tslint -t stylish '{path}'\""
```

Comment:
`tslint` is [missing](https://palantir.github.io/tslint/usage/cli/) the `--force` parameter. We do not want to crash the pipeline if linting fails during dev mode